### PR TITLE
Merge 6.x -> main

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by Magisus** [comment]
Looks like we had some testing around the checksums that is now failing.

**Comment by Magisus** [comment]
I think we're waiting to cut a 7 release branch before this gets merged.

**Comment by Magisus** [comment]
Release branch has been created, this can be merged once Travis is green.

**Review by justinstoller — APPROVED** [review]

**Pull request merged**
